plugins/wdc/wdc-nvme.c

```diff
@@ -9648,10 +9648,10 @@ static int wdc_fetch_log_file_from_device(struct nvme_dev *dev, __u32 fileId,
 					  __u16 spiDestn, __u64 fileSize, __u8 *dataBuffer)
 {
 	int ret = WDC_STATUS_FAILURE;
-	__u32                     chunckSize = WDC_DE_VU_READ_BUFFER_STANDARD_OFFSET;
-	__u32                     maximumTransferLength = 0;
-	__u32                     buffSize = 0;
-	__u64                     offsetIdx = 0;
```

Comment:
Please do not mix whitespace changes with code changes.

Reply:
Sorry for mixing the changes everytime so just split the changes to commits.